components/navbar/ChainSelectDropdown.vue

```diff
@@ -42,5 +42,13 @@ export default class ChainSelect extends Vue {
     this.$store.dispatch('setUrlPrefix', value)
     this.$router.push({ path: `/${value}` })
   }
+
+  get chainName() {
+    if (this.selected === 'rmrk') {
+      return 'kusama'
+    } else {
+      return this.selected
+    }
+  }
```

Comment:
Let's use `getChainNameByPrefix` function in `utils/chain.ts`.

Reply:
done, thanks